### PR TITLE
Remove retired members from the store

### PR DESCRIPTION
**Pull request by jumaffre**
Part of #1401 

Retired members are now removed the store instead of being marked as `Retired`. We also remove all their associated data in the store, including their governance history.

The renamed `remove_member` proposal is also idempotent.

### CODE REVIEW

File: doc/governance/accept_recovery.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ When the recovery threshold is reached, the ``POST recovery_share`` RPC returns
 
 Once the recovery of the private ledger is complete on a quorum of nodes that have joined the new network, the ledger is fully recovered and users are able to continue issuing business transactions.
 
-.. note:: Recovery shares are updated every time a new recovery member is added or retired and when the ledger is rekeyed. It also possible for members to update the recovery shares via the ``update_recovery_shares`` proposal.
+.. note:: Recovery shares are updated every time a new recovery member is added or remove and when the ledger is rekeyed. It also possible for members to update the recovery shares via the ``update_recovery_shares`` proposal.
```
Comment (achamayou) [inline doc/governance/accept_recovery.rst:80]:
```suggestion
.. note:: Recovery shares are updated every time a new recovery member is added or removed and when the ledger is rekeyed. It also possible for members to update the recovery shares via the ``update_recovery_shares`` proposal.
```

File: python/ccf/clients.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ class Response:
 
     def __str__(self):
         versioned = (self.view, self.seqno) != (None, None)
-        status_color = "red" if self.status_code / 100 in (4, 5) else "green"
+        status_color = "red" if int(self.status_code / 100) in (4, 5) else "green"
```
Comment (achamayou) [inline python/ccf/clients.py:184]: Is status code now a string?

**Review by achamayou — APPROVED** [review]

File: python/ccf/clients.py (hunk 2 above)
Comment (eddyashton) [reply python/ccf/clients.py:184]: I think this is just
```suggestion
        status_color = "red" if self.status_code // 100 in (4, 5) else "green"
```

Comment (jumaffre) [reply python/ccf/clients.py:184]: This used to be a float division, which only worked for `400` and `500`. I didn't know about `//` which is neater!

**Pull request merged**